### PR TITLE
Specify mi-go beam message

### PR DESCRIPTION
**Pull request by casswedson**
#### Summary
None

#### Purpose of change
I don't like how the game says "the "mi-go name" cast mi-go slaver beam
at you" doesn't seem in line with the spooky scary mi-gos it seems like
someone wanted to hide all spell messages by adding the SILENT flag to
the extra spell casted but that flag does other things, it seems one
can't hide all monster spell messages at this moment

#### Describe the solution
Give them a more specific message, something that does not mention
spells or casting anything at you

#### Testing
man-migo

### CODE REVIEW

File: data/json/monsters/mi-go.json [hunk 1]
```diff
@@ -83,7 +83,12 @@
     "special_attacks": [
       [ "PARROT", 800 ],
       { "id": "scratch", "damage_max_instance": [ { "damage_type": "cut", "amount": 23, "armor_multiplier": 0.8 } ] },
-      { "type": "spell", "spell_data": { "id": "mi-go_slaver_beam", "min_level": 3 }, "cooldown": 100 }
+      {
+        "type": "spell",
+        "spell_data": { "id": "mi-go_slaver_beam", "min_level": 3 },
+        "cooldown": 100,
+        "monster_message": "The mi-go's numerous antennae sync their movement only for a moment."
```
Comment (Inglonias) [inline data/json/monsters/mi-go.json:90]: Minor editorial change. This is purely down to personal preference on my part. Feel free to ignore:

`The mi-go's numerous antennae sync their movement for a brief moment.`

Comment (casswedson) [reply data/json/monsters/mi-go.json:90]: Oh yeah no, I'd say I am really bad at descriptions and text in genral, anything is welcome

**Comment by Amoebka** [comment]
... but the mi-go slaver beam isn't a dazing scream produced by the antennae? The descriptions of all mi-go that can beam mention them holding weapons.
Slaver: " It is carrying an oblong object that hums with an odd keening sound."
Myrmidon: "It actually appears to be carrying weaponry."
Guard: "several appendages terminating in what appear to be devices of some sort"
It appears to be some sort of a gun they shoot, not something they do naturally (except the guard, who has it melded into its hands?)

**Comment by casswedson** [comment]
My intent was to just get rid of an unintended message, not at all explain what is doing what, I don't think is good to outright or indirectly explain what the things the mi-go have with them or are part of their bodies are for. "Oh yeah they fire pain beams with *this* or *that* alien device"
>Slaver: " It is carrying an oblong object that hums with an odd keening sound."

They have plenty of natural abilities (like shock, pain beam etc) let your imagination decide what does what, don't explain
>Myrmidon: "It actually appears to be carrying weaponry."

Has an actual alien gun, this bit of description refers to that
>Guard: "several appendages terminating in what appear to be devices of some sort"

See my response for the slaver

**Comment by Amoebka** [comment]
Myrmidon doesn't have an actual alien gun that does damage, only scouts get those. At any rate, my issue is that "sync their antennae" really gives you no clue as to what it is doing. I'm not saying you should "explain" how it works, but you should at least make it clear in the message itself that the pain beam is being used.

Even something simple like "Mi-go slaver shoots a pain beam at you" would remove the reference to spellcasting, but also be much clearer that the proposed message.

**Review by Maleclypse — APPROVED** [review]

**Pull request merged**